support-frontend/assets/pages/contributions-landing/contributionsLandingReducer.ts

```diff
@@ -63,6 +63,7 @@ export interface StripeCardFormData {
 	formComplete: boolean;
 	setupIntentClientSecret: string | null;
 	recurringRecaptchaVerified: boolean;
+	// TODO: No non-serialisable values should be in Redux state!! This needs to be refactored
```

Comment:
Oh it's a bit of a disaster how much non-serialisable state is in this reducer

Reply:
Well at least now we're going to have a lot of angry red reminders in the console during development about how bad it is 😂